### DISCUSSION TITLE
How do I maximize the Query Results pane, or open it in a / detach it to a new tab?

### DISCUSSION THREAD

**Discussion by manojbaishya** [microsoft/azuredatastudio#18720] [Q&A]
As stated in the title.

I do not see the option in the grid context-menu, command palette or keyboard shortcuts.

![image](https://user-images.githubusercontent.com/28330014/158110789-92ac4a61-ee5d-44d0-92ae-e4d087920a56.png)

**Comment by kburtram** [comment]
We currently don't support this directly, other than resizing the results window by dragging the sash divider.  Though we are beginning work on https://github.com/microsoft/azuredatastudio/issues/4784, which will allow the results to be displayed in a separate tab.

**Reply by gregclerk** [reply]
Ticket [#4784](https://github.com/microsoft/azuredatastudio/issues/4784) is for displaying results in a separate **TAB** but how about the option to display in a separate **WINDOW** so results could be on an entirely different monitor?
Thanks

**Reply by erinstellato-ms** [reply]
@gregclerk Confirming I understand your request, which would require the ability to un-dock a window in ADS and move it to another monitor.  Correct?

**Reply by gregclerk** [reply]
Not the whole tab but just the bottom Results section similar to how Visual Studio can have the Command Prompt section undocked and moved to a different monitor altogether.  
The now-moved Results window would be wired into all open tabs so that whenever a query is run from any tab the results will be displayed in this now-moved Results window.
Thanks

**Comment by erinstellato-ms** [comment]
This functionality is dependent on VS Code adding the ability to undock a section/window/tab.  This is a requested item in VS Code that you can upvote, with details about the effort required for implementation here: https://github.com/microsoft/vscode/issues/10121#issuecomment-345770248
